neurobooth_os/tasks/task_sidetrials.py

```diff
@@ -0,0 +1,79 @@
+from neurobooth_os.tasks import Task_Eyetracker, utils
+
+
+class Task_sidetrials(Task_Eyetracker):
+    """
+    Tasks with two trials, one for each 'side', dominant or not dominant. It supports fixation tasks with side-trials,
```

Comment:
Nit: change comment to "motor" tasks - this should never have been in fixation task in the first place. Thanks for fixing this!